data/data/coreos/rhcos.json

```
@@ -1,107 +1,107 @@
 {
   "stream": "rhcos-4.16",
```

Can you make this change? Our automated script doesn't do it.

```suggestion
  "stream": "rhcos-4.17",
```

---

done, I have changed it.